CNAME

```diff
@@ -1 +0,0 @@
-axpdsp.org
```

Comment:
Are we moving hosting out of github pages / will keep this domain?

Reply:
We are keeping the hosting on github pages. The CNAME file is automatically generated in the gh-pages branch by the build action. I was planning on keeping the existing domain name, that will need to be changed in build.yaml after it is merged.

Reply:
gotcha 👍